src/main/java/com/redislabs/redisai/RedisAI.java

```diff
@@ -350,6 +367,49 @@ public boolean runScript(String key, String function, String[] inputs, String[]
     }
   }
 
+  /**
+   * {@code AI.SCRIPTRUN <key> <function> [TIMEOUT t] INPUTS <input> [input ...] [$ input ...]
+   * OUTPUTS <output> [output ...]}
+   */
+  public boolean runScript(
```

Comment:
@sazzad16 WDYT of using `public boolean runScript(String key, String function, String[] inputs, String[] outputs, boolean variadicInputs)` given the current available method is `public boolean runScript(String key, String function, String[] inputs, String[] outputs)` ?

Reply:
@filipecosta90 I don't mind. It is just a matter of preference. I prefer to keep everything about a parameter together (if possible).

Here, both `inputs` and `variadicInputs` collectively represent INPUTS argument. So I preferred it this way.